src/RecipeCard.js

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
I love how you made a separate component for the recipe card. Your code looks very organized. The naming of file that way makes it very clear what this component is about.